src/book/02-system/02-subsystems/06-logging.mdx

```diff
@@ -0,0 +1,42 @@
+---
+section: System
+chapter: Subsystems
+title: Logging
+description: Logging and how it works in the NUbots codebase.
+slug: /system/subsystems/logging
+---
+
+# Data Playback
+
+## Description
+
+DataPlayback is a module which will allow you to play back `.nbs` files as if they were happening in realtime. This can be very useful for when you are trying to diagnose a problem for which you have a recording as you can continuously play back the same data.
```

Comment:
```suggestion
The DataPlayback module will allow you to play back `.nbs` files as if they were happening in realtime. This can be very useful when you are trying to diagnose a problem for which you have a recording since you can continuously play back the same data.
```